dbt/adapters/spark/impl.py

```diff
@@ -138,13 +143,54 @@ def quote(self, identifier: str) -> str:  # type: ignore
     def _get_relation_information(self, row: agate.Row) -> RelationInfo:
         """relation info was fetched with SHOW TABLES EXTENDED"""
```

Comment:
@Fokko : Could you extend the docstring to explain that the `SHOW TABLES EXTENDED` is preferred because fetching multiple tables at once is faster than fetching tables one by one. And, that we except the downside of parsing the ` |---` string given the performance gains?